Footer

lmorelli986 opened the pull request:
modificato stile dell'header in vista desktop

lmorelli986 committed:
corretto label del copy link all'interno della navbar relativa al titolo della sezione

lmorelli986 committed:
stile header: stilizzata area del document title a sinistra (icgILiPt)

lmorelli986 committed:
stile header: stilizzata area del form di ricerca in linea (icgILiPt)

lmorelli986 committed:
stile header: stilizzata area della selezione versione (icgILiPt)

lmorelli986 committed:
stile header: stilizzata area quicklinks a destra (icgILiPt)

lmorelli986 committed:
stilizzato footer come su io.italia.net (4sqFBdsp)

pdavide committed:
Update package-lock.json

pdavide committed:
Update package-lock.json

pdavide committed:
Update markup and styles

pdavide reviewed (changes requested):
- L'header non è visualizzato correttamente su mobile, dovrebbe essere come nel mockup.
- Il footer non è visualizzato correttamente su mobile (sistemare spaziatura tra gli elementi).

lmorelli986 committed:
creata progressbar dello scroll di pagina (JtVNBtoM)

lmorelli986 committed:
interventi multipli:
- creato layout mobile del footer
- integrato modifiche di pulizia codice di header e footer
- integrato traduzioni i10n

pdavide commented on docs_italia_theme/layouts/footer.html:
```diff
@@ -1,62 +1,67 @@
-        <div class="col-12 pt-3">
-          <ul class="footer-menu list-inline pl-0">
-            {% for link in site.data.footer_links %}
-              {% set link_name = link.title %}
```
Si possiamo assumerlo, perché si tratta di elementi del tema che non possono essere modificati da chi userà il tema per i propri documenti.

lmorelli986 replied:
Ok perfetto

lmorelli986 committed:
stile etichetta in consultazione inclinata anche in mobile

lmorelli986 committed:
fix del footer mobile e stilizzato come io.italia.it

lmorelli986 committed:
reso dinamico l'accesso alle immagini nel footer tramite direttiva "pathto"

lmorelli986 committed:
fix del layout dell'header in ambiente mobile

lmorelli986 committed:
aggiunta mock delle entry del bottone versioni dell'header

lmorelli986 committed:
Revert "stile etichetta in consultazione inclinata anche in mobile"

This reverts commit d56b9a8b434b537419f18cc5f34823776e428dd9.

pdavide committed:
Better footer responsiveness

- assets cleanup
- style and markup updates
